photos/db.go

```diff
@@ -0,0 +1,478 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Spencer Kimball (spencer.kimball@gmail.com)
+
+package main
+
+import (
+	"database/sql"
+	"errors"
+	"math/rand"
+	"net/url"
+	"time"
+
+	"github.com/cockroachdb/cockroach/security"
+	"github.com/cockroachdb/cockroach/util"
+	"github.com/cockroachdb/cockroach/util/log"
+	_ "github.com/lib/pq"
+)
+
+const (
+	lgSchema = `
+CREATE DATABASE IF NOT EXISTS photos;
+
+CREATE TABLE IF NOT EXISTS photos.users (
+  id           INT,
+  photoCount   INT,
+  commentCount INT,
+  name         STRING,
+  address      STRING,
+
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS photos.photos (
+  id           BYTES DEFAULT EXPERIMENTAL_UUID_V4(),
+  userID       INT,
+  commentCount INT,
+  caption      STRING,
+  latitude     FLOAT,
+  longitude    FLOAT,
+  timestamp    TIMESTAMP,
+
```

Comment:
Do you think we should also add in a blob to represent the photo, and the thumbnail? Maybe later